File: Célia_LingProgram/JavaFacul/src/Lista02/Exer05_IMC.java
package Lista02;

import java.util.Scanner;

public class Exer05_IMC {
    public static void main(String[] args) {

        Double altura, peso, imc;
        Integer sexo;

        Scanner leitor = new Scanner(System.in);

        System.out.println("Indice de Massa Corporal");
        System.out.println("Insira sua altura:");
        altura = leitor.nextDouble();
        System.out.println("Insira se peso:");
        peso = leitor.nextDouble();
        System.out.println("Insira seu sexo: \n" +
                "1- Feminino. \n" +
                "2- Masculino.");
        sexo = leitor.nextInt();

        imc = peso /(altura * 2);

        switch (sexo){
            case 1:
            if (imc < 19.1) {
                System.out.println("Abaixo do peso");
            } else if (imc > 19.1 && imc < 25.8) {
                System.out.println("No peso normal");
            } else if (imc > 25.8 && imc < 27.3) {
                System.out.println("Marginalmente acima do peso");
            } else if (imc > 27.3 && imc < 32.3) {
                System.out.println("Acima do peso ideal");
            } else {
                System.out.println("Obeso");
            }
            break;

            case 2:

            if (imc < 20.7) {
                System.out.println("Abaixo do peso");
            } else if (imc > 20.7 && imc < 26.4) {
                System.out.println("No peso normal");
            } else if (imc > 26.4 && imc < 27.8) {
                System.out.println("Marginalmente acima do peso");
            } else if (imc > 27.8 && imc < 31.1) {
                System.out.println("Acima do peso ideal");
            } else {
                System.out.println("Obeso");
            }
            break;

            default:
            System.out.println("Selecione um dos Sexos Válidos");
            break;

        }

        System.out.println("Seu imc é: "+ imc);
    }
}

File: Célia_LingProgram/ListaJavaFacul/src/L2_Exer05_IMC/Exer05_IMC.kt
package Lista02

import java.util.*

//Crie uma classe chamada “IndiceMassaCorporal” que:
//A. Solicite a altura e peso do usuário.
//B. Solicite o sexo, deve ser:
//1 - Mulher
//2 - Homem.
//C. Calcule o IMC do usuário.
//D. Exiba a Condição de saúde da pessoa baseada em seu IMC.
//Use os valores desse site: http://www.lilianspeziali.com.br/tabela-de-imc/

fun main() {
    val reader = Scanner(System.`in`)
    println("Indice de Massa Corporal")
    println("Insira sua altura: ")
    val altura: Double = reader.nextDouble()
    println("Insira seu peso: ")
    val peso: Double = reader.nextDouble()
    println("___________________\n" +
            "1  | Mulher\n" +
            "2  | Homem")
    println("Insira o código referente ao seu sexo: ")
    val sexo: Int = reader.nextInt()

    val imc = peso / (altura * 2)
    println( """___________________
    Resultado: """)

    when (sexo) {
        1 -> when {
            imc < 19.1 -> println("Abaixo do peso.")
            imc < 25.8 -> println("No peso normal.")
            imc < 27.3 -> println("Marginalmente acima do peso.")
            imc < 32.3 -> println("Acima do peso ideal.")
            else -> {
                println("Obeso.")
            }
        }
        2 -> when {
            imc < 20.7 -> println("Abaixo do peso.")
            imc < 26.4 -> println("No peso normal.")
            imc < 27.8 -> println("Marginalmente acima do peso.")
            imc < 31.1 -> println("Acima do peso ideal.")
            else -> {
                println("Obeso.")
            }
        }
    }
}